content/_index.md

```diff
@@ -56,9 +56,6 @@ The principles that Matrix attempts to follow are:
         the global Matrix network
     -   Fully open standard - publicly documented standard with no IP or
         patent licensing encumbrances
-    -   Fully open source reference implementation - liberally-licensed
-        example implementations with no IP or patent licensing
-        encumbrances
```

Comment:
![image](https://github.com/user-attachments/assets/4d4ca622-5d44-4958-84c5-d7d2ff691bf3)

Reply:
Oh, where is this from?

Reply:
It's from the foundation's rules (Guiding Principles) https://matrix.org/media/2019-06-10%20-%20Matrix.org%20Foundation%20CIC%20Rules.pdf

Reply:
Hm, but then there are no reference implementations in existence today? I'm not sure if the foundation still considers this as one of its future goals. If it does, I guess we can also keep the part of the spec that mentions reference implementations.

Reply:
Regardless of the *aspirations* of the foundation, there are currently no reference implementations (of either clients or servers), so the spec should not claim that there are.

Reply:
This is the list of
> The principles that Matrix attempts to follow

Are they not "aspirations"?

Reply:
the text says: "In particular, it is envisaged that the Foundation will undertake the following activities:" - i.e. these are aspirations as written in 2018. Other details there (e.g. referring to an "mx:// URI scheme") are also wrong. The bug here is that the Rules haven't been updated since first being conceived, and are overdue being updated to reflect how the Foundation's scope has evolved (by the guardians in consultation with the governing board).

Reply:
I am referring to https://github.com/Johennes/matrix-spec/blob/083e6ef25da712eecc1b56a8e3a2fafaf48d130d/content/_index.md?plain=1#L48

It seems incorrectly phrased in the spec then and should also have been adjusted with this PR. Or perhaps this list should not be here at all in favor of a link to the document Gnuxie cited.

Reply:
Ugh, I hadn't realised that the heading of this list was "The principles that Matrix attempts to follow are:". I thought this bit of the spec was just a statement of fact, rather than a list of what the Matrix project is *trying* to achieve. I guess I should have read more carefully. (In my defence, I'd rather bias to action than agonise over every change.)

But it does make this PR less obviously correct. If time and money were unlimited, would the Matrix Foundation nowadays seek to maintain liberally-licensed reference implementations? Probably? It's not obvious (eg, W3C does not maintain reference implementations for HTML.) It's an open question, to be resolved by the Matrix guardians and governing board. 

I guess it feels like this whole section of the spec is a bit outdated and out-of-place. (AFAICT, it was written in 2014 and has been barely touched since.) I'd rather the spec were stating the current fact, and the hopes and dreams of the matrix project were written elsewhere.

Not entirely sure how best to proceed here, but I'm inclined to take time to think about it rather than rush anything.